openapi.yaml

```diff
@@ -595,7 +595,8 @@ components:
       description: ID of meal plan
       required: true
       schema:
-        type: string
+        type: number
```

Comment:
Why changing to number? ID must be a string base on JSONAPI: https://jsonapi.org/format/#document-resource-object-identification

Reply:
Good catch. meal-plan IDs are stored as integers in oracledb so I added a pattern instead like with `osuId`